docs/build/getting-started/multicall.md

```diff
@@ -107,8 +116,8 @@ function blockAndAggregate(Call[] calldata calls)
 
 
 ### Function: aggregate3
-- ## Description:
-This function aggregates calls similar to aggregate but allows specifying whether failures are allowed for each call individually.
+- #### Description:
```

Comment:
```suggestion
#### Description:
```
or 
```suggestion
* **Description**:
```

Whichever you prefer, but please apply it to the others, as well as the `Return Values`